.github/CONTRIBUTOR_GUIDE/ct/AppName.md

```diff
@@ -0,0 +1,231 @@
+# **AppName<span></span>.sh Scripts**
```

Comment:
```suggestion
# AppName.sh Scripts
```

I am only going to put this comment once in this review, but as far as I know the following results in the same render:
```
# **AppName.sh Scripts**
# AppName.sh Scripts
```

Thus I would like to propose to keep the markdown clean and not add the formatting.

Reply:
I put <span> there so it will not work as a Link.